platforms/substrate/configuration/deploy-network.yaml

```diff
@@ -55,7 +55,6 @@
       name: "{{ playbook_dir }}/../../shared/configuration/roles/setup/scripts"
     vars:
       namespace: "{{ org.name | lower }}-subs"
-      network_type: "{{ network.type | lower }}"
```

Comment:
Saw this change in other files as well. Did we change anything related to network.type or was this not used ?

Reply:
This is not used